freqtrade/configuration/arguments.py

```diff
@@ -119,3 +124,11 @@ def _build_subcommands(self) -> None:
         )
         list_exchanges_cmd.set_defaults(func=start_list_exchanges)
         self._build_args(optionlist=ARGS_LIST_EXCHANGES, parser=list_exchanges_cmd)
+
+        # Add download-data subcommand
+        download_data_cmd = subparsers.add_parser(
+            'download-data',
+            help='Download backtesting data.'
+        )
+        download_data_cmd.set_defaults(func=start_download_data)
+        self._build_args(optionlist=ARGS_DOWNLOADER, parser=download_data_cmd)
```

Comment:
Imho it's better to use a unique schema for naming these optionlists -- so ARGS_DOWNLOAD_DATA would be better. Just imho.